pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -654,6 +654,16 @@ func DefaultOpenAPIConfig(config configapi.MasterConfig) *openapicommon.Config {
 			},
 		}
 	}
+	if config.OAuthConfig != nil {
+		securityDefinitions["Oauth2AccessToken"] = &spec.SecurityScheme{
+			SecuritySchemeProps: spec.SecuritySchemeProps{
+				Type:             "oauth2",
+				Flow:             "accessCode",
+				AuthorizationURL: config.OAuthConfig.MasterURL + "/oauth/authorize",
```

Comment:
Actually I thought about this some more and we should be able to use the variables.  We just need to make sure that the Swagger generation script always starts a server on localhost.